src/Pokeview/components/Type.scss

```diff
@@ -0,0 +1,122 @@
+@mixin type-content {
+    height: 60%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 0.5em;
+    font-size: 1.1vw;
```

Comment:
I recommend you to use em instead of percentage units

Reply:
fixed